Bottom: c3c95597a157fcea6aa3cbdacf6b26631247389b
Top:    ac1a57ca92b5fde95dfda767f022d7afa3d2b0d9
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-18 16:09:22 -0800

Refresh of af-ib-conn

---

diff --git a/src/acm.c b/src/acm.c
index 187125f..08036ad 100755
--- a/src/acm.c
+++ b/src/acm.c
@@ -64,18 +64,6 @@ static pthread_mutex_t acm_lock = PTHREAD_MUTEX_INITIALIZER;
 static int sock = -1;
 static short server_port;
 
-struct ib_connect_hdr {
-	uint8_t  cma_version;
-	uint8_t  ip_version; /* IP version: 7:4 */
-	uint16_t port;
-	uint32_t src_addr[4];
-	uint32_t dst_addr[4];
-#define cma_src_ip4 src_addr[3]
-#define cma_src_ip6 src_addr[0]
-#define cma_dst_ip4 dst_addr[3]
-#define cma_dst_ip6 dst_addr[0]
-};
-
 static int ucma_set_server_port(void)
 {
 	FILE *f;
